pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -3863,3 +3881,8 @@ func Test_queuedPodInfo_gatedSetUponCreationAndUnsetUponUpdate(t *testing.T) {
 		t.Error("Expected pod to be ungated")
 	}
 }
+
+func newAttemptedQueuedPodInfo(podInfo *framework.QueuedPodInfo) *framework.QueuedPodInfo {
```

Comment:
```suggestion
func attemptQueuedPodInfo(podInfo *framework.QueuedPodInfo) *framework.QueuedPodInfo {
```

because this function is not creating a new object.